src/router/index.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable radix */
+
+'use strict';
+
+const express = require('express');
+const db = require('./../db');
+const products = require('./../db/products.json');
+
+const router = express.Router();
+
+// router.get('/', productsRoutes.allProducts);
+
+// creating a route - route gets all product
+router.get('/', (req, res) => res.json(products));
+
+
+// registerting a new user
+router.post('/register', (req, res, next) => {
+  console.log('New registeration in process');
+  db.register(req.body);
+  res.sendStatus(200);
+  next();
+});
+
+// logging in
+router.post('/login', (req, res, next) => {
+  res.sendStatus(200);
+  next();
+});
+
+//  get single product
+router.get('/db/products/:id', (req, res, next) => {
+  const found = products.some(product => product.id === console.log(parseInt(req.params.id)));
```

Comment:
Does this in-line console.log() work? I would remove it and put any logging you want on a separate line.

Reply:
Actually that in-line console log didn't work i was just testing something out forgot to take it out. It has been taken off in part 3.